src/app/shared/issue-tables/issue-tables.component.html

```diff
@@ -106,11 +106,18 @@
         <mat-icon>link</mat-icon>
       </button>
       <button *ngIf="permissions.isTeamResponseEditable() && !issue.status
-       && this.isActionVisible(action_buttons.RESPOND_TO_ISSUE)"
+       && this.isActionVisible(action_buttons.RESPOND_TO_ISSUE); else tryFixIssue"
               [routerLink]="'issues/' + issue.id" mat-button color="accent"
               style="transform: scale(0.8)" matTooltip="Respond to this Issue">
         <mat-icon>feedback</mat-icon>
       </button>
+      <ng-template #tryFixIssue>
+        <button *ngIf="this.isActionVisible(action_buttons.FIX_ISSUE)"
+                mat-button color="accent"
+                style="transform: scale(0.8)" matTooltip="Fix this Issue" >
+          <mat-icon>edit</mat-icon>
```

Comment:
I've actually re-used the icon that is currently being used for the faulty issues table in Team's Response Phase.
Should we change the tool tip to "Edit this Issue" ?

Reply:
Yes sure, changing to edit instead of fix sounds reasonable.

Reply:
Minor point: I noticed that we have capitalised the word "issue" in some tooltips, but not capitalised it in others. Should we standardise?

For example, `Delete this Issue`, `Edit this Issue` or `Delete this issue`, `Edit this issue`

Reply:
Sure! 👍

Reply:
I'll tackle this in a separate PR?

Reply:
Okay, I have opened an issue for it. https://github.com/CATcher-org/CATcher/issues/344